internal/pkg/pkg.go

```diff
@@ -150,6 +154,15 @@ func (p *Pkg) String() string {
 	return string(p.DisplayPath)
 }
 
+// RelativePathTo returns current package's path relative to a given package.
```

Comment:
Explain the semantics of the output string. It's OS-specific path (not slash seperated).